src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -218,6 +230,72 @@ private List<string> GetExamples(XPathNavigator nav, ITripleSlashCommentParserCo
             return GetMultipleExampleNodes(nav, "/member/example").ToList();
         }
 
+        private void ResolveCodeSource(XDocument doc, ITripleSlashCommentParserContext context)
+        {
+            foreach (XElement node in doc.XPathSelectElements("//code"))
+            {
+                var source = node.Attribute("source");
+                if (source == null)
+                {
+                    continue;
+                }
+
+                var region = node.Attribute("region");
+
+                var path = source.Value;
+                if (!Path.IsPathRooted(path))
+                {
+                    path = Path.Combine(context.Source?.BasePath ?? ".", path);
```

Comment:
sorry, my question might not be clear. i think you should use context.Source.Path, which means current comments' src file path. Currently BasePath is not used.